src/featherlight.js

```diff
@@ -525,7 +525,7 @@
 				} else if(fl.persist !== false) {
 					$target.data('featherlight-persisted', fl);
 				}
-				if (elemConfig.$currentTarget.blur) {
+				if (elemConfig.$currentTarget) {
```

Comment:
`$currentTarget` should be a jQuery set, so the previous test is silly. I'm not sure your replacement is needed, but can't hurt.

Reply:
@marcandre it is probably an edge case, but IE11 cannot run the if condition if elemConfig.$currentTarget is NULL and this prohibits featherlight from closing the modal correctly.

Reply:
I understand, but in what circumstances would `$currentTarget` be `null`?

Reply:
I have to re-check if the issue still persists, however it was the case in IE11 on some specific elements. We had created this workaround for a customer specific project.